use-timescale/metrics-logging/service-metrics.md

```diff
@@ -89,6 +92,48 @@ increase your storage capacity, or reduce the size of your database. When you
 have done that, you can also add a retention policy, or turn on compression, to
 avoid the problem occurring again in the future.
 
+## Query-level statistics in the Timescale dashboard
+
+You can analyze your queries by navigating to the `Query stats` tab from the
+Services dashboard.
+
+<EarlyAccess />
+
+Use the filter at the top of the page to view details of your queries, including
+how many rows the query returned, and the time the query took to run.
+
+<img class="main-content__illustration"
+src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc_querystats_expanded.webp"
+width={1375} height={944}
+alt="The Timescale query stats viewer showing the statistics for a recent query" />
+
+In the results window, use the horizontal scroll bar at the bottom of the screen
+to see statistics for your query. You can also use the arrow to the left of the
+query to expand it. The columns show these details about your queries:
+
+|Column|Description|
+|-|-|
+|Executions|Number of times the query has been run during the results period|
+|Total rows|Total number of rows scanned, inserted, or updated by the query during the results period|
+|Median rows|Median number of rows touched each time the query is run|
+|P05 time|The fifth percentile, or the minimum time this query took|
+|Median time|Median number of microseconds (µs) or milliseconds (ms) it took to run the query|
+|P95 time|The ninety-fifth percentile, or the maximum time this query took|
+|Median memory|Median amount of memory consumed by the query
+|P95 memory|The ninety-fifth percentile, or the maximum memory this query used|
+|Median bytes read|Median number of bytes used each time the query is run|
+|P95 bytes read|The ninety-fifth percentile, or the maximum number of bytes this query used|
+|Cache hit ratio|Total sum of memory used, as a percentage|
```

Comment:
The cache hit ratio metric is the ratio between the memory (cache) and disk usage. We can describe them as the cache hit / (cache hit + cache miss). It shows how often the data needed for the query was available in memory.

Could we change the description? 

Could we change the description?